eclib/Makefile

```diff
@@ -8,25 +8,34 @@ CHECKS   ?= jasmin
 # --------------------------------------------------------------------
 DESTDIR  ?=
 PREFIX   ?= /usr/local
-BINDIR   := $(PREFIX)/bin
 LIBDIR   := $(PREFIX)/lib
-SHRDIR   := $(PREFIX)/share
 INSTALL  ?= install
 
 # --------------------------------------------------------------------
-.PHONY: default usage check install uninstall
+.PHONY: default usage check install uninstall clean
 
 default: check
 
 usage:
-	@echo "Usage: make <target> where <target> in [check|install|uninstall]" >&2
+	@echo "Usage: make <target> where <target> in [check|clean|install|uninstall]" >&2
 
 check:
 	easycrypt runtest $(ECARGS) $(ECCONF) $(CHECKS)
 
-install:
-	$(INSTALL) -m 0755 -d $(DESTDIR)$(LIBDIR)/jasmin/easycrypt
-	$(INSTALL) -m 0644 -t $(DESTDIR)$(LIBDIR)/jasmin/easycrypt *.ec
+jasmin.conf:
+	@echo "# DO NOT MODIFY" | tee $@
+	@echo "# This file is generated and may be silently overwritten" | tee -a $@
+	@echo "[general]" | tee -a $@
+	@echo "idirs=Jasmin:$(LIBDIR)/easycrypt/jasmin" | tee -a $@
+
+install: jasmin.conf
+	$(INSTALL) -m 0755 -d $(DESTDIR)$(LIBDIR)/easycrypt/jasmin
+	$(INSTALL) -m 0644 -t $(DESTDIR)$(LIBDIR)/easycrypt/jasmin *.ec
+	$(INSTALL) -m 0755 -d $(DESTDIR)$(LIBDIR)/easycrypt/config
+	$(INSTALL) -m 0644 -t $(DESTDIR)$(LIBDIR)/easycrypt/config *.conf
 
 uninstall:
 	$(RM) -r $(DESTDIR)$(LIBDIR)/jasmin
```

Comment:
This should be updated also I guess. Apart from that, I can merge.

Reply:
Done. Thanks for the careful review.